crates/re_log_types/src/lib.rs

```diff
@@ -205,6 +205,27 @@ impl std::fmt::Display for ApplicationId {
 
 // ----------------------------------------------------------------------------
 
+/// Options for configuring viewer behavior when a blueprint is ready.
+#[derive(Copy, Clone, Debug, PartialEq, Eq)] // `PartialEq` used for tests in another crate
+#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
+pub struct BlueprintReadyOpts {
+    /// Make this the active blueprint for the `app_id`.
+    pub make_active: bool,
+
+    /// Make this the default blueprint for the `app_id`.
```

Comment:
We should probably expand on this a bit.

For instance: what happens if there is no active blueprint and I send one with `make_active: false`? Does it make it active anyway?

Reply:
Agreed -- will add some more description.

In short: if this blueprint becomes the default, then it **might** become the active blueprint if there is not already an active blueprint. Make_active=false would not prevent that from happening. :thinking: